4.7.3 fails to build with latest utf8-string

Build failure:

```
[13 of 21] Compiling Data.ListLike.UTF8 ( src/Data/ListLike/UTF8.hs, /tmp/ListLike-4.7.3/dist-newstyle/build/x86_64-linux/ghc-8.10.3/ListLike-4.7.3/build/Data/ListLike/UTF8.o, /tmp/ListLike-4.7.3/dist-newstyle/build/x86_64-linux/ghc-8.10.3/ListLike-4.7.3/build/Data/ListLike/UTF8.dyn_o )

src/Data/ListLike/UTF8.hs:153:10: error:
    • Overlapping instances for IsString (UTF8 BSC.ByteString)
        arising from the superclasses of an instance declaration
      Matching instances:
        instance UTF8.UTF8Bytes string index => IsString (UTF8 string)
          -- Defined in ‘Data.String.UTF8’
        instance IsString (UTF8 BSC.ByteString)
          -- Defined at src/Data/ListLike/UTF8.hs:150:10
    • In the instance declaration for
        ‘StringLike (UTF8 BSC.ByteString)’
    |
153 | instance StringLike (UTF8 BS.ByteString) where
    |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Data/ListLike/UTF8.hs:277:10: error:
    • Overlapping instances for IsString (UTF8 BSLC.ByteString)
        arising from the superclasses of an instance declaration
      Matching instances:
        instance UTF8.UTF8Bytes string index => IsString (UTF8 string)
          -- Defined in ‘Data.String.UTF8’
        instance IsString (UTF8 BSLC.ByteString)
          -- Defined at src/Data/ListLike/UTF8.hs:274:10
    • In the instance declaration for
        ‘StringLike (UTF8 BSLC.ByteString)’
    |
277 | instance StringLike (UTF8 BSL.ByteString) where
    |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

So constrained bounds or a new release would fix this! 

Thanks :)

--- a/src/Data/ListLike/UTF8.hs
+++ b/src/Data/ListLike/UTF8.hs
@@ -31,7 +31,9 @@ import Data.ListLike.String (StringLike(..))
 import Data.Maybe (fromMaybe)
 import Data.Monoid (Monoid(..))
 import Data.Semigroup (Semigroup(..))
+#if !MIN_VERSION_utf8_string(1,0,2)
 import Data.String (IsString(fromString))
+#endif
 import Data.String.UTF8 (UTF8{-, UTF8Bytes-})
 import qualified Data.String.UTF8 as UTF8
 import GHC.Exts (IsList(..))
@@ -147,8 +149,10 @@ instance ListLikeIO (UTF8 BS.ByteString) Char where
     -- writeFile = BS.writeFile
     -- appendFile = BS.appendFile
 
+#if !MIN_VERSION_utf8_string(1,0,2)
 instance IsString (UTF8 BS.ByteString) where
     fromString = UTF8.fromString
+#endif
 
 instance StringLike (UTF8 BS.ByteString) where
     toString = UTF8.toString
@@ -271,8 +275,10 @@ instance ListLikeIO (UTF8 BSL.ByteString) Char where
 instance Semigroup (UTF8 BSL.ByteString) where
   (<>) = mappend
 
+#if !MIN_VERSION_utf8_string(1,0,2)
 instance IsString (UTF8 BSL.ByteString) where
     fromString = UTF8.fromString
+#endif
 
 instance StringLike (UTF8 BSL.ByteString) where
     toString = UTF8.toString

